tasks/internal/pulp-push-disk-images/pulp-push-disk-images.yaml

```diff
@@ -42,7 +42,7 @@ spec:
       description: Success if the task succeeds, the error otherwise
   steps:
     - name: pull-and-push-images
-      image: quay.io/konflux-ci/release-service-utils:6556e8a6b031c1aad4f0472703fd121a6e1cd45d
+      image: quay.io/konflux-ci/release-service-utils:597145c2cbcff9e99e1c62169eccab003e68c157
```

Comment:
Can you update the tests to use the same? Otherwise LGTM.

Reply:
done, sorry about that

Reply:
no worries